server/app/services/program/ActiveAndDraftPrograms.java

```diff
@@ -57,6 +57,28 @@ public static ActiveAndDraftPrograms buildFromCurrentVersionsUnsynced(
     return new ActiveAndDraftPrograms(repository, Optional.empty(), allProgramTypes);
   }
 
+  /**
+   * Queries the existing active and draft versions of disabled programs and builds a snapshotted
+   * view of their state. Like its counterpart, this does not include question definitions due to
+   * the absence of ProgramService.
+   */
+  public static ActiveAndDraftPrograms buildFromCurrentVersionsUnsyncedDisabledProgram(
```

Comment:
**nitpick:** Thoughts on naming this something like `buildDisabledProgramsFromCurrentVersionsUnsynced`? I think having DisabledProgram as part of the name reads a bit better. (And same with naming below)

Reply:
Yeah no problem, changed the name on this one and the one below.